### PR TITLE
chore: remove unused pub/sub from docker compose

### PR DESCRIPTION
**Pull request by efgpinto**
Refs #1456 

~Opening as draft: Just looking to confirm the setup we want before doing the changes on all samples. Would this be ok?~

### CODE REVIEW

File: samples/java-customer-registry-quickstart/docker-compose.yml [hunk 1]
```diff
@@ -1,18 +1,21 @@
+# If you're looking to use eventing with Google PubSub, to get an emulator running:
+# - uncomment property "-Dkalix.proxy.eventing.support=google-pubsub-emulator"
+# - uncomment the env var PUBSUB_EMULATOR_HOST and the section below for gcloud-pubsub-emulator service
```
Comment (octonato) [inline samples/java-customer-registry-quickstart/docker-compose.yml:3]: This is looking good. I think it's all that we need.

**Review by johanandren — APPROVED** [review]
LGTM (did not in any way cross-check which samples may use eventing)

**Comment by efgpinto** [comment]
> LGTM (did not in any way cross-check which samples may use eventing)

Ahm, actually, I think I need to double check that.. I might have overlooked eventing.in.

**Comment by octonato** [comment]
> > LGTM (did not in any way cross-check which samples may use eventing)
> 
> Ahm, actually, I think I need to double check that.. I might have overlooked eventing.in.

Anyone with a proto using the topic annotations. Should be easy to find it out.

**Pull request merged**